.github/workflows/helm-chart-ci.yaml

```diff
@@ -114,8 +106,6 @@ jobs:
 
     needs: [lint-chart]
 
-    if: needs.lint-chart.outputs.changed == 'true'
```

Comment:
I was thinking on running with something like this.

```
if: needs.lint-chart.outputs.changed == 'true' || needs.diff.outputs.workflow-changed || needs.diff.outputs.ci-scripts-changed || needs.diff.outputs.tests-changed
```

or maybe more simple a single output from a diffs job.

```
if: needs.lint-chart.outputs.changed == 'true' || needs.diff.outputs.requires-test-run
```

That way it only runs when required.